app/templates/dependencies/eslint/.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+    "rules": {
+        "indent": [
+            2,
+            4
+        ],
+        "quotes": [
+            2,
+            "single"
+        ],
+        "linebreak-style": [
+            2,
+            "unix"
+        ],
+        "semi": [
+            2,
+            "always"
+        ],
+        "no-console": 1
+    },
+    "env": {
+        "node": true,
+        "browser": true
```

Comment:
I set both to true to enable lint on public/js files.  Maybe I'll remove it out of the .eslintrc and can be specified on the documentation.